Assets/Plugins/Appboy/Editor/PostBuild.cs

```diff
@@ -5,8 +5,8 @@
 using UnityEngine;
 using UnityEditor;
 using UnityEditor.Callbacks;
-using UnityEditor.iOS.Xcode;
-using UnityEditor.iOS.Xcode.Extensions;
+using UnityEditor.iOS.Xcode.Custom;
+using UnityEditor.iOS.Xcode.Custom.Extensions;
```

Comment:
Hi @vietphan90vn,

Thanks for submitting this PR. Rather than bundling a custom Xcode `dll`, I'd prefer to just gate the incompatible method calls using macros (e.g. `#if UNITY_4_0`) - what do you think about that approach?

Rather than using a custom dll, I'd prefer if we just gated the

Reply:
My approach work for Unity 5+ project. Your mean is copy XCode API source code to "plugins" folder and adds macro define?

Reply:
Hi @vietphan90vn, I'm thinking about using platform dependent compilation, e.g.:

```
#if UNITY_2017_3_OR_NEWER
   AddFileToEmbedFrameworks("path to library X");
   AddFileToEmbedFrameworks("path to library Y");
   AddFileToEmbedFrameworks("path to library Z");
#else
   Debug.Log("Automatically adding libraries to Xcode embedded frameworks not supported in this version of Unity. Remember to add libraries XYZ to embedded frameworks in your Xcode project");
#endif
```

If you'd rather stick to your working solution, feel free to continue using your fork as well.

Reply:
OK, you don't want support Unity 5 to auto adding libraries to Xcode embedded frameworks. I don't want to stick my solution, just want to help the Unity project with the version before 2017. I close this pull request, thanks for your feedback.

Reply:
Ok, You don't want to support Unity 5 projects anymore. I just try to help other using Unity 5 project (cannot update to Unity2017) like as me. I close this pull request.